### DISCUSSION TITLE
Is it possible to use Synpress to test trust wallet?

### DISCUSSION THREAD

**Discussion by philpetrov** [Synthetixio/synpress#930] [General]
Hi guys! I use cypress and synpress as a plugin for testing metamask. Just as many people know, the second popular wallet is trust wallet. Can I somehow use this synpress library to connect and use trust wallet in cypress tests?

**Comment by duckception** [comment]
Hey! 

Unfortunately, this is not possible unless you gather the selectors yourself and make a wrapper class for that wallet. 

We're working on a new release of Synpress that will allow you to easily plug in such wrapper class to your tests, however, the current version does not support that so you'd have to handle this yourself in your codebase.
